utils/scarb-ui/src/args/verbosity.rs

```diff
@@ -100,29 +122,33 @@ mod tests {
                 verbose: 0,
                 quiet: 0,
                 verbosity: Some(level),
+                no_warnings: false
             }),
             level
         );
     }
 
-    #[test_case(2, 0, Verbosity::Quiet, tracing_core::LevelFilter::OFF)]
-    #[test_case(1, 0, Verbosity::Quiet, tracing_core::LevelFilter::OFF)]
-    #[test_case(0, 0, Verbosity::Normal, tracing_core::LevelFilter::ERROR)]
-    #[test_case(0, 1, Verbosity::Verbose, tracing_core::LevelFilter::WARN)]
-    #[test_case(0, 2, Verbosity::Verbose, tracing_core::LevelFilter::INFO)]
-    #[test_case(0, 3, Verbosity::Verbose, tracing_core::LevelFilter::DEBUG)]
-    #[test_case(0, 4, Verbosity::Verbose, tracing_core::LevelFilter::TRACE)]
-    #[test_case(0, 5, Verbosity::Verbose, tracing_core::LevelFilter::TRACE)]
+    #[test_case(2, 0, false, Verbosity::Quiet, tracing_core::LevelFilter::OFF)]
+    #[test_case(1, 0, false, Verbosity::Quiet, tracing_core::LevelFilter::OFF)]
+    #[test_case(0, 0, false, Verbosity::Normal, tracing_core::LevelFilter::ERROR)]
```

Comment:
Why we do not need to change this tests, if we do: 
```
impl From<VerbositySpec> for Verbosity {
    fn from(spec: VerbositySpec) -> Self {
        match spec.integer_verbosity() {
            v if v < -1 => Verbosity::Quiet,
            -1 => Verbosity::NoWarnings,
```
Shouldn't `quiet=1, verbose=0` be the new one then? 🤔

Reply:
I'm not sure I understand, `quiet=1, verbose=0` shouldn't produce any different verbosity level than before? That's not something this PR aims to change 🤔

Reply:
But you are changing `impl From<VerbositySpec> for Verbosity` which is used to parse it, right?

Reply:
I am, but only to create conditions for setting `NoWarnings` verbosity level, without affecting impact of `--quiet` and `--verbose` flags on settings their respectful verbosity level.